projects/angular/src/data/datagrid/datagrid-detail.spec.ts

```diff
@@ -53,25 +54,25 @@ export default function (): void {
         expect(context.clarityElement.innerHTML).not.toContain(content);
       });
 
-      it('hides content with the esc key', async(() => {
+      it('hides content with the esc key', waitForAsync(() => {
```

Comment:
```suggestion
      it('hides content with the esc key', async(() => {
```

This is another unrelated change. Please revert. It will be fixed when #1440 is merged.

Reply:
Done